src/handlers/poap.ts

```diff
@@ -0,0 +1,14 @@
+import { rainbowFetch } from '../rainbow-fetch';
+import { parsePoaps } from '@rainbow-me/parsers';
+import logger from 'logger';
+
+export const fetchPoaps = async (address: string) => {
+  try {
+    const url = `https://api.poap.xyz/actions/scan/${address}`;
+    const data = await rainbowFetch(url, {});
```

Comment:
I'd use a way shorter timeout (something between 5 and 10), default is 30. (this will delay ALL the NFTs to be loaded, displayed, rendered)